$NetBSD: patch-configure,v 1.1 2013/08/26 15:00:33 fhajny Exp $

Make piddir configurable.
Cross-compiling: assume snprintf terminates long strings (IRIX's does).

--- configure.orig
+++ configure
@@ -1888,7 +1888,7 @@
 	echo $ac_n "checking whether snprintf correctly terminates long strings""... $ac_c" 1>&6
 echo "configure:1890: checking whether snprintf correctly terminates long strings" >&5
 	if test "$cross_compiling" = yes; then
-    { echo "configure: error: can not run test program while cross compiling" 1>&2; exit 1; }
+  echo "$ac_t""yes (assumed: cross-compiling)" 1>&6
 else
   cat > conftest.$ac_ext <<EOF
 #line 1895 "configure"
@@ -3397,14 +3397,6 @@
 fi
 
 
-# make sure the directory exists
-if test ! -d $piddir ; then	
-	piddir=`eval echo ${sysconfdir}`
-	case $piddir in
- 		NONE/*) piddir=`echo $piddir | sed "s~NONE~$ac_default_prefix~"` ;;
-	esac
-fi
-
 cat >> confdefs.h <<EOF
 #define _PATH_SPREAD_PIDDIR "$piddir"
 EOF
